src/models/giveaway.js

```diff
@@ -0,0 +1,17 @@
+import mongoose from 'mongoose'
+
+import mongooseService from '../services/mongoose'
+
+const schema = new mongoose.Schema({
+  name: String,
+  numberOfEditions: Number,
+
+  // @TODO: Figure out provisioning story
+  metadata: {
```

Comment:
Since Giveaways won't be created until after their metadata is created, this can be a `required` field.

Reply:
Tomorrow show me how to retrieve the metadata document itself instead of just the ID. we need to grab the image from it to display on the collection screen.